src/models/Room.ts

```diff
@@ -0,0 +1,24 @@
+import { Schema, model } from 'mongoose';
+
+interface Schedule {
+    dateOfWeek: string;
+    start: string;
+    end: string;
+}
+
+interface Room {
+    name: string;
+    schedule: [Schedule];
+    closed: boolean;
+}
+
+const schedule = {} as Schedule;
+const roomSchema = new Schema<Room>({
+    name: { type: String, required: true },
+    schedule: [schedule],
```

Comment:
Does this work properly? We might want to explicitly describe what a schedule looks like in the schema for mongoose